apps/web/playwright/event-types.e2e.ts

```diff
@@ -367,9 +367,99 @@ testBothFutureAndLegacyRoutes.describe("Event Types tests", () => {
         await unCheckDisplayLocation(page);
       });
     });
+    test("Can add single use links", async ({ page }) => {
+      await gotoFirstEventType(page);
+
+      // Go to the advanced tab.
+      await page.getByTestId("vertical-tab-event_advanced_tab_title").click();
+
+      // Check if single-use link doesn't exist initially.
+      await expect(page.locator("[data-testid=remove-single-use-link-0]")).toHaveCount(0);
+
+      await addSingleUseLink(page);
+
+      // Save the event.
+      await saveEventType(page);
+
+      // Check if the single-use link exists.
+      await expect(page.locator("[data-testid=remove-single-use-link-0]")).toBeVisible();
+    });
+    test("Can remove single use links", async ({ page }) => {
+      await gotoFirstEventType(page);
+
+      // Go to the advanced tab.
+      await page.getByTestId("vertical-tab-event_advanced_tab_title").click();
+
+      // Check if single-use link doesn't exist initially.
```

Comment:
Have you test this locally?

Reply:
@anikdhabal Actually yes, I was about to ask this question only - Should I refactor this to match the new toggle layout, or will we remove this test.

As I was not confirmed that if we will keep this or not, therefore I didn't modify it. But yes this will fail currently, I am aware.

Reply:
Please refactor it and add the test after completing the all task. Here you need to add test for both frontend and backend.

Reply:
@anikdhabal Got it. Just to reiterate to check if I didn't misunderstood anything -

- Only when the task is completed, then only I will refactor this test.
- I have already added the part, where the it is making a booking on the front-end and after that checking if the single-use links is no more present on the backend. So, is there anything else that needs to be added. This covers both front-end and backend right?

Reply:
Pls refactor the whole test for now. Pls add test after completing the functionality and when all things are up. I will guide you that time

Reply:
@anikdhabal Sorry for asking again, but did you mean "remove" or "refactor" in this line -

> Pls refactor the whole test for now

Reply:
I mean remove your changes

Reply:
Thanks @anikdhabal Done!